README.md

```diff
@@ -36,5 +24,11 @@ You can also check more details at [Vision SDK FAQ](https://vision.mapbox.com/#f
 
 ## Installation and setup
```

Comment:
Could you please also statement that `Vision*` directories are deprecated?

Reply:
Or even remove all current content except for Examples description, and add a short separate note - `MapboxVision*` directories are deprecated and will be removed soon